.github/workflows/go.yml

```diff
@@ -98,7 +104,7 @@ jobs:
             matrix:
                 go:
                     - 1.18.10
-                    - 1.21.6
+                    - 1.22.0
```

Comment:
Ideally we would just use the same matrix version values for all platforms like you are suggesting. I use 1.18/1.22 for regular linux because I want to test against the latest 1.18/1.22 versions, which I can do by passing 1.18/1.22 to actions/setup-go. In this case, setup-go determines the latest 1.18/1.22 versions and downloads them. However for Amazon linux I cannot use actions/setup-go, so I need to manually download Go, which requires a URL that includes the full version path. For example, I use `wget https://go.dev/dl/go1.18.10.linux-amd64.tar.gz`. If I just used 1.18 in the URL it would download 1.18.0. I could change the regular linux to use full versions, but then we'd have to manually change the patch version if we wanted to test a newer version. Let me know what you think.

Reply:
I see
May be it is good enough to use 1.18.10 on all platforms

Reply:
We could do that, but if we want to use matrix.go in the makefile target name (eg 'install-tools-go1.18.10') then whenever we want to update the patch version we'd have to update it in go.yml and in the makefile target name